packages/sanity/src/hydration-test/server.tsx

```diff
@@ -1,33 +1,31 @@
-import 'typescript-support';
 import path from 'path';
 import glob from 'glob';
 import {WebpackConfigurator, runTestsInPuppeteer, serve, IServer} from 'ui-autotools-utils';
-import {renderMetadata} from './import-and-render';
+import {renderMetadata} from './render-metadata';
 
 const packageDir = path.resolve(__dirname, '..');
 const projectDir = process.cwd();
-const metaGlob = 'src/**/*.meta.ts?(x)';
 const webpackConfigPath = path.join(projectDir, 'meta.webpack.config.js');
 
-function getWebpackConfig(ssrComps: string[]) {
+function getWebpackConfig(ssrComps: string[], metaGlob: string) {
   return WebpackConfigurator
     .load(webpackConfigPath)
-    .setEntry('meta', glob.sync(path.join(projectDir, metaGlob)))
-    .addEntry('meta', path.join(packageDir, 'ssr-renderer', 'test-page.js'))
-    .addEntry('meta', path.join(packageDir, 'ssr-test', 'index.js'))
+    .setEntry('meta', glob.sync(metaGlob))
+    .addEntry('meta', path.join(packageDir, 'hydration-test', 'test-page.js'))
```

Comment:
Couldn't you require the test page in the index file instead of adding it as an entry?